cob_bringup/robots/base_solo.launch

```diff
@@ -16,19 +16,9 @@
 	<!-- start robot_state_publisher -->
 	<node pkg="robot_state_publisher" type="state_publisher" name="robot_state_publisher" />
 
-	<group unless="$(arg use_old_base_drive_chain)" >
-		<!-- start base driver -->
-		<include file="$(find cob_bringup)/drivers/canopen_402.launch" >
-			<arg name="robot" value="$(arg robot)" />
-			<arg name="component_name" value="base" />
-			<arg name="can_device" value="can0" />
-		</include>
-
-		<!-- start base controller -->
-		<include file="$(find cob_bringup)/controllers/base_controller_plugin.launch" >
-			<arg name="robot" value="$(arg robot)" />
-		</include>
-	</group>
+	<include file="$(find cob_bringup)/components/canopen_base.launch" unless="$(arg use_old_base_drive_chain)">
+		<arg name="robot" value="$(arg robot)"/>
+	</include>
```

Comment:
shouldn't the `group` block below be exchanged with the `legacy_base.launch` file`?